### DISCUSSION TITLE
Upload torch saved model to wandb

### DISCUSSION THREAD

**Discussion by talhaanwarch** [Lightning-AI/pytorch-lightning#14414] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I have trained a model and after training, i saved the model. now i want to upload the saved .pt file to wandb. How can i do that. I check this documentation https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.loggers.wandb.html but i cant find the option

**Comment by turian** [comment]
Just create a [ModelCheckpoint](https://pytorch-lightning.readthedocs.io/en/stable/common/checkpointing_intermediate.html) and pass that as a callback to trainer.

**Comment by rohitgr7** [comment]
did you check the `log_model`parameter of `WandbLogger`?
https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.loggers.wandb.html#pytorch_lightning.loggers.wandb.WandbLogger.params.log_model

**Reply by talhaanwarch** [reply]
i am saving my model like this way
`torch.save(model.model.state_dict(), 'model.pt')` but i dont know how to log this model. I dont want to log checkpoints

**Reply by talhaanwarch** [reply]
i think i got this, i can save it as wandb_logger.experiment.save('model.pt')
